aiagents4pharma/talk2biomodels/states/state_talk2biomodels.py

```diff
@@ -12,13 +12,14 @@ class Talk2Biomodels(AgentState):
     """
     The state for the Talk2BioModels agent.
     """
-    model_id: Annotated[list, operator.add]
-    # sbml_file_path: str
+    llm_model: str
     # A StateGraph may receive a concurrent updates
     # which is not supported by the StateGraph.
     # Therefore, we need to use Annotated to specify
     # the operator for the sbml_file_path field.
     # https://langchain-ai.github.io/langgraph/troubleshooting/errors/INVALID_CONCURRENT_GRAPH_UPDATE/
+    model_id: Annotated[list, operator.add]
     sbml_file_path: Annotated[list, operator.add]
     dic_simulated_data: Annotated[list[dict], operator.add]
-    llm_model: str
+    dic_scanned_data: Annotated[list[dict], operator.add]
+    dic_steady_state_data: Annotated[list[dict], operator.add]
```

Comment:
Is there a reason to have `dic_simulated_data`, `dic_scanned_data`, and `dic_steady_state_data`, instead of a single `dic_simulated_data` struc?. From my understanding, these all have similar shape (i.e., time vs concentration) and could be differentiated by the experiment_id key?

Reply:
Based on our conversation over Teams, this can be dealt with in this PR or a separate PR: up to you.